pkg/config/config.go

```diff
@@ -197,58 +190,12 @@ func (c WebConfig) Validate() (err error) {
 
 // Validate that the TRISA config has mTLS certificates for operation.
 func (c *TRISAConfig) Validate() error {
-	if c.Pool == "" || c.Certs == "" {
-		return errors.New("invalid configuration: specify pool and cert paths")
+	if c.Certs == "" {
+		return errors.New("invalid configuration: specify certificates path")
 	}
 	return nil
 }
 
-// LoadCerts returns the mtls TRISA trust provider for setting up an mTLS 1.3 config.
-// NOTE: this method is not thread-safe, ensure it is not used from multiple go-routines
-func (c *TRISAConfig) LoadCerts() (_ *trust.Provider, err error) {
-	if c.certs == nil {
-		if err = c.load(); err != nil {
-			return nil, err
-		}
-	}
-	return c.certs, nil
-}
-
-// LoadPool returns the mtls TRISA trust provider pool for creating an x509.Pool.
-// NOTE: this method is not thread-safe, ensure it is not used from multiple go-routines
-func (c *TRISAConfig) LoadPool() (_ trust.ProviderPool, err error) {
-	if len(c.pool) == 0 {
-		if err = c.load(); err != nil {
-			return nil, err
-		}
-	}
-	return c.pool, nil
-}
-
-// Load and cache the certificates and provider pool from disk.
-func (c *TRISAConfig) load() (err error) {
-	var sz *trust.Serializer
-	if sz, err = trust.NewSerializer(false); err != nil {
-		return err
-	}
-
-	if c.certs, err = sz.ReadFile(c.Certs); err != nil {
-		return fmt.Errorf("could not parse certs: %w", err)
-	}
-
-	if c.pool, err = sz.ReadPoolFile(c.Pool); err != nil {
-		return fmt.Errorf("could not parse cert pool: %w", err)
-	}
-	return nil
-}
-
-// Reset the certs cache to force load the pool and certs again
-// NOTE: this method is not thread-safe, ensure it is not used from multiple go-routines
-func (c *TRISAConfig) Reset() {
-	c.pool = nil
-	c.certs = nil
-}
-
```

Comment:
This functionality was duplicated for the `TRISAConfig` and the `TRPConfig` -- it's decoupled with `MTLSConfig` and the use of an embedding.